src/libraries/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/Http2Connection.cs

```diff
@@ -226,6 +226,7 @@ public async ValueTask SetupAsync(CancellationToken cancellationToken)
                 // Processing the incoming frames before sending the client preface and SETTINGS is necessary when using a NamedPipe as a transport.
                 // If the preface and SETTINGS coming from the server are not read first the below WriteAsync and the ProcessIncomingFramesAsync fall into a deadlock.
                 _ = ProcessIncomingFramesAsync();
+                LogExceptions(InitialSettingsReceived.Task);
```

Comment:
InitialSettingsReceived is lazily initialized, but this would seem to effectively always force it into existence. Also the exception for this task is explicitly set in only one function as far as I can tell. From my perspective, if we want to suppress the unobservedness, that should be done just after setting the exception; logging it if desired can also be done there.